### PR TITLE
Updated the 'Installing Kubernetes with Kubespray'

### PR DESCRIPTION
**Pull request by ydFu**
1. Add the OS supported by the current version of Kubespray.
   [ref 1: kubespray](https://github.com/kubernetes-sigs/kubespray/blob/master/README.md#deploy-a-production-ready-kubernetes-cluster)

2. Supplement the features provided by the Kubespray overview.
   [ref 2: kubespray](https://github.com/kubernetes-sigs/kubespray/blob/master/README.md#deploy-a-production-ready-kubernetes-cluster)

3. Update Supported Linux Distributions.
   [ref 3: Supported Linux Distributions](https://github.com/kubernetes-sigs/kubespray#supported-linux-distributions)

4. Improve the description in '(1/5) Meet the underlay requirements'
   [ref 4: Requirements](https://github.com/kubernetes-sigs/kubespray/blob/master/README.md#requirements)

5. Add whatsnext.

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubespray.md [hunk 1]
```diff
@@ -6,21 +6,27 @@ weight: 30
...
-* support for most popular Linux distributions
-  * Ubuntu 16.04, 18.04, 20.04, 22.04
-  * CentOS/RHEL/Oracle Linux 7, 8
-  * Debian Buster, Jessie, Stretch, Wheezy
-  * Fedora 34, 35
-  * Fedora CoreOS
-  * openSUSE Leap 15
-  * Flatcar Container Linux by Kinvolk
-* continuous integration tests
+This quickstart helps to install a Kubernetes cluster hosted on **AWS, GCE, Azure, OpenStack, vSphere, Equinix Metal(bare metal), Oracle Cloud Infrastructure (Experimental), or Baremetal** with [Kubespray](https://github.com/kubernetes-sigs/kubespray).
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/tools/kubespray.md:9]: pls wrap the long lines like this?

File: content/en/docs/setup/production-environment/tools/kubespray.md [hunk 2]
```diff
@@ -6,21 +6,27 @@ weight: 30
...
+  - **Ubuntu** 16.04, 18.04, 20.04, 22.04
+  - **CentOS/RHEL** 7, 8
+  - **Fedora** 34, 35
+  - **Fedora CoreOS**
+  - **openSUSE** Leap 15.x/Tumbleweed
+  - **Oracle Linux** 7, 8
+  - **Alma Linux** 8
+  - **Rocky Linux** 8
+  - **Amazon Linux 2** 
+* **Continuous integration tests**
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/tools/kubespray.md:29]: Why are we adding bold format to these texts?

Comment (ydFu) [reply content/en/docs/setup/production-environment/tools/kubespray.md]: With reference to the original kubernetes-sigs/kubespray annotation, it has been adjusted
 [ref 4: Requirements](https://github.com/kubernetes-sigs/kubespray/blob/master/README.md#requirements)

File: content/en/docs/setup/production-environment/tools/kubespray.md (hunk 1 above)
Comment (ydFu) [reply content/en/docs/setup/production-environment/tools/kubespray.md]: Fix it.

**Comment by windsonsea** [comment]
thanks.
/lgtm

**Pull request merged**